### ISSUE TITLE
Tab min button should have an aria-label

### ISSUE THREAD

**Issue by jackieleighh** [caplin/FlexLayout#83]
Hi!  The min button - `button.flexlayout__tab_toolbar_button-min` - needs an aria-label to pass accessibility checks.  Thanks!
